Title: Print pkg-config Conf File to Stdout
DEP: 3
State: ACCEPTED
Date: 2015-01-22
Drivers: Christopher Reichert <creichert07@gmail.com>
URL: http://hackage.haskell.org/package/hsqml
Source: http://anonscm.debian.org/darcs/pkg-haskell/haskell-hsqml
Abstract:
 Prints the pkg-config conf file name to stdout. The output is
 expected by the cdbs tool.
--- a/Setup.hs
+++ b/Setup.hs
@@ -288,6 +288,7 @@ regWithQt pkg@PackageDescription { libra
     Just regFile -> do
       writeUTF8File (fromMaybe (display (packageId pkg) <.> "conf") regFile) $
         I.showInstalledPackageInfo instPkgInfo'
+      putStrLn $ fromMaybe (display (packageId pkg) <.> "conf") regFile
     _ | fromFlag (regGenScript flags) ->
       die "Registration scripts are not implemented."
       | otherwise ->
